src/system/CategoriesModule/Resources/views/Form/filter_layout_customizations.html.twig

```diff
@@ -1,10 +0,0 @@
-{% extends 'bootstrap_4_horizontal_layout.html.twig' %}
-{# this template is used in the blocks module #}
-
-{% block form_label_class -%}
-    col-md-3
-{%- endblock form_label_class %}
-
-{% block form_group_class -%}
-    col-md-9
-{%- endblock form_group_class %}
```

Comment:
this appears to be completely removed and not replaced?

Reply:
yes; not needed anymore, because it is identical to https://github.com/zikula/core/blob/master/src/Zikula/FormExtensionBundle/Resources/views/Form/bootstrap_4_zikula_admin_layout.html.twig#L3-L9